### DISCUSSION TITLE
What plugin adds the "run" and "debug" after text and how do I use them?

### DISCUSSION THREAD

**Discussion by ijsnow** [LazyVim/LazyVim#2239] [Q&A] [answered]
I installed my nvim setup on a new personal computer and opened up a rust side project and noticed these "run" and "debug" "after text" things. I can't figure out the plugin that adds them and would like to see what its like to use them! Could someone clue me in? Thanks!

<img width="266" alt="Screenshot 2023-12-23 at 12 58 02 PM" src="https://github.com/LazyVim/LazyVim/assets/11557758/5752e387-ad00-4a86-ad63-85c370b52c78">

**Comment by dpetka2001** [comment, marked as answer]
Should be `rust-tools`.

**Reply by ijsnow** [reply]
Ah so it’s part of the runnables/debuggables? Thanks!
